daemon/src/engine/framework/ConsoleHistory.h

```diff
@@ -31,18 +31,40 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #ifndef FRAMEWORK_CONSOLE_HISTORY_H_
 #define FRAMEWORK_CONSOLE_HISTORY_H_
 
+#include <string>
+#include <vector>
+#include <mutex>
```

Comment:
Avoid including system headers all over the place. We include all the system headers in q_shared.h, which is included by Common.h, which is built as a precompiled header and included everywhere.